packages/web-app-admin-settings/src/components/Users/ContextActions.vue

```diff
@@ -0,0 +1,62 @@
+<template>
```

Comment:
I think we can refactor this component to be more generic in the future - at least for the admin settings in general.